### DISCUSSION TITLE
Is there a ballpark timeline for when the MVP might be released in most browsers?

### DISCUSSION THREAD

**Discussion by spencerkohan** [gpuweb/gpuweb#3541] [Q&A] [answered]
I found this [old thread](https://github.com/gpuweb/gpuweb/issues/1276) which was pointing towards a possible 2022 release date.  As an observer it seems maybe a bit farther away - for instance Safari preview no longer has an option to turn on WebGPU in developer mode - but it is hard to judge progress from the outside.

Is there any sense of when the MVP might ship in the major browsers?  I realize there may not be clear timelines established at this stage, but even a broad outlook - i.e. will this be a year, a couple years, 5+ years etc. would be a huge help in terms of understanding where to allocate effort.

Thanks in advance for any clarity and insights which can be provided!

**Comment by kdashg** [comment, marked as answer]
Expect 2023!

**Reply by spencerkohan** [reply]
Cool!

**Reply by arbassic** [reply]
Can't wait to start the development of a brand new game engine :)

**Comment by puxiao** [comment]
https://chromiumdash.appspot.com/schedule
**113**
**Stable Release  Tue, May 2, 2023**
